pycparser/__init__.py

```diff
@@ -15,7 +15,7 @@
 from .c_parser import CParser
 
 
-def preprocess_file(filename, cpp_path='cpp', cpp_args=''):
+def preprocess_file(filename, cpp_path='cpp', cpp_args='', **kwargs):
```

Comment:
Thinking about it for a bit, this may be premature generalization. For example seeing kwargs in parse_file is unnatural since that function does a number of different things. I think we can just add an "encoding" field with a default value of None for now?